fastfield_codecs/src/bitpacked.rs

```diff
@@ -16,27 +16,32 @@ pub struct BitpackedReader {
     bit_unpacker: BitUnpacker,
     pub min_value_u64: u64,
     pub max_value_u64: u64,
+    pub num_vals: u64,
 }
 
-impl FastFieldCodecReader for BitpackedReader {
+impl FastFieldCodecDeserializer for BitpackedReader {
```

Comment:
I'd put that in the `XXXXSerializer`s and rename the trait `XXXXCodec`.

Reply:
I think don't fully understand. I split the Deserializer from the `FastFieldDataAccess`, because e.g. in the merge case we provide access without deserializing.